src/cmd/manage_nsfs.js

```diff
@@ -257,6 +258,15 @@ async function fetch_bucket_data(argv, from_file) {
         fs_backend: data.fs_backend || undefined
     };
 
+    // fs_backend deletion specified with empty string '' (but it is not part of the schema)
+    if (argv.fs_backend === '') {
```

Comment:
line 258 will do the same in the object assignment, so this is redundant, you can still keep the comment if you wish
I suggest also to do the same for s3_policy if possible

Reply:
I will remove this and add the comment for `s3_policy`, thanks.